src/allEqual.js

```diff
@@ -0,0 +1,28 @@
+import { pipe, uniq } from 'ramda';
+
+import lengthLte from './lengthLte';
+
+// Original idea for this function was conceived by https://github.com/jackmellis
+// in https://github.com/char0n/ramda-adjunct/pull/513.
+
+/**
+ * Checks that all elements in a list are equal.
```

Comment:
I think you need to note what 'equal' means in this context i.e. 'Checks that all elements in a list are equal using `R. uniq` which uses `R.equals` for comparison'.

Reply:
Will explain it explicitly.